source/reporting/index.html.md.erb

```diff
@@ -246,9 +246,9 @@ For example, to list all the payments your PSP sent to you on 16 June 2020:
 GET /v1/payments?from_settled_date=2020-06-16T00:00:00Z&to_settled_date=2020-06-16T23:00:00Z
 ```
 
-These take inputs based on a subset of the [ISO 8601 standard](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard). For example, a valid input would be `2015-08-13T12:35:00Z`.
+These query parameters take inputs based on a subset of the [ISO 8601 standard](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard). For example, a valid input would be `2015-08-13T12:35:00Z`.
 
-If a payment has not happened yet, the payment will not be in the response.
+If a payer has not yet sent a payment, the payment will not be in the response.
```

Comment:
This disambiguation is clearly useful but I believe the "payment has not happened yet" in this context refers to the Stripe payment to your bank account. 

In that case I think this might read something like 

 ```suggestion
If your PSP has not sent this payment to your bank account yet, the payment will not be in the response.
```

I'm not sure, as with the other content it's tricky because of the overloaded "payment"